src/fetcher/user/request.ts

```diff
@@ -20,15 +20,15 @@ export async function requestUser(
   })
 
   if (uuid.__typename === 'User') {
-    // TODO: Need to be implemented in the API
+    // stub: Need to be implemented in the API
```

Comment:
what do you mean by stub?

Reply:
but yes, note is enough here. already has an issue:
https://github.com/serlo/frontend/issues/935

Reply:
Stub in the sense of mock / placeholder implementation